login/login.go

```diff
@@ -29,6 +31,13 @@ func main() {
 	}
 	loadUsers(file)
 
+	if domain != "" {
+	    customerUrl = "http://accounts." + domain + "/customers/search/findByUsername"
```

Comment:
should use formatting, but this will be re-written in https://github.com/weaveworks/weaveDemo/pull/103